packages/server/resources/views/emails/email_verification.edge

```diff
@@ -0,0 +1,21 @@
+<mjml>
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏What resources did you use to learn MJML?

Reply:
I paid someone on fiver !
Seriously I've just look at existing templates https://mjml.io/templates